final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +75,68 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
-
+scary_monster = {
+  'name' => 'Deforestation',
+  'smell' => 'cedar',
+  'weight' => 3000,
+  'citiesDestroyed' => ['Seattle', 'Denver', 'Portland'],
+  'luckyNumbers' => [13, 77, 99],
+  'address' => {
+    "number" => 1810,
+    "street" => "Julian",
+    "state" => "Colorado",
+    "zip" => 80204
+    }
+  }
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
-
+#   - energy_level = 50
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  attr_reader :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age)
+    self.name = name
+    self.super_power = super_power
+    self.age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+  end
+
+  def say_name
+    p name
```

Comment:
You don't just want to print any name here- the self.name, or @name will be needed to print the right thing.